tests/fdb/api/test_auxiliary.cc

```diff
@@ -49,20 +57,54 @@ std::set<eckit::PathName> setup(FDB& fdb) {
     key.set("levtype", "sfc");
     key.set("param", "130");
 
-    key.set("step", "1");
-    fdb.archive(key, data, length);
 
-    key.set("date", "20111213");
-    fdb.archive(key, data, length);
+    // key.set("step", "1");
+
+    // fdb.archive(key, data, length);
+
+    // key.set("date", "20111213");
+    // fdb.archive(key, data, length);
 
-    key.set("type", "pf");
-    fdb.archive(key, data, length);
+    // key.set("type", "pf");
+    // fdb.archive(key, data, length);
+
+    for (const auto& date : dates) {
+        key.set("date", date);
+        for (const auto& type : types) {
+            key.set("type", type);
+            for (const auto& step : steps) {
+                key.set("step", step);
+                fdb.archive(key, data, length);
+            }
+        }
+    }
 
     fdb.flush();
 
     return auxPaths;
 }
 
+void expectEmpty(eckit::PathName dir) {
+
+    // If this directory exists, the wipe did not complete. Print some useful diagnostic info...
+    if (dir.exists()) {
+        std::cout << "err -- expected dir to no longer exist: " << dir << std::endl;
```

Comment:
cout or cerr?

Reply:
Or better, use the eckit logging infrastructure. We are in an eckit environment.